plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -126,6 +126,9 @@
     private Duration fileStatusCacheExpireAfterWrite = new Duration(1, MINUTES);
     private long fileStatusCacheMaxSize = 1000 * 1000;
     private List<String> fileStatusCacheTables = ImmutableList.of();
+
+    // TODO: Change this to Optional.empty() along with other such Trino migration changes
+    private Optional<String> alternateViewCodecTag = Optional.of("Presto");
```

Comment:
We should accept our own views out of the box.
I don't see why would create burden for the users.